_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/assembly/assembly--product-try-it-hero.html.twig

```diff
@@ -4,10 +4,26 @@
      <div class="product-try-it">
       {{ content.field_title }}
       {{ content.field_content }}
-      <a href="{{ content.field_learn_more_link.value }}">Learn More <span class="fa fa-caret-right"></span></a>
-      <div class="product-try-it-cta">
-        <a class="button heavy-cta" href={{ content.field_try_it_link.value }}><span class="fa fa-bell"></span> Try It</a>
-      </div>
+      {% if content.field_learn_more_link|render %}
+        <a href="{{ content.field_learn_more_link.0['#url'] }}">
+        {% if content.field_learn_more_link.0['#title'] %}
+          {{ content.field_learn_more_link.0['#title'] }}
+        {% else %}
+        Learn More
+        {% endif %}
+        &nbsp;<span class="fa fa-caret-right"></span></a>
+      {% endif %}
+      {% if content.field_learn_more_link|render %}
```

Comment:
Did you mean to check `content.field_try_it_link` here instead of `content.field_learn_more_link`?